src/components/IsUnlocked.js

```diff
@@ -61,8 +66,27 @@ const defaultState = {
   inputValue: {
     password: '',
   },
+  incorrectPassword: false,
 }
 
+export const PageTitle = styled(Box).attrs({
+  width: '152px',
+  height: '27px',
+  fontFamily: 'MuseoSans',
+  fontSize: '22px',
+  color: '#1d2028',
```

Comment:
prefers using theme colors variables

Reply:
I took colors from the designs, will look for corresponding ones in the theme